android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/MetadataResolver.java

```diff
@@ -90,13 +96,13 @@ public static void findUrlMetadata(final Context context, final MetadataResolver
    *
    * @param url The url for which to request data
    */
-  private static void requestUrlMetadata(String url) {
+  private static void requestUrlMetadata(String url, int txPower, int rssi) {
```

Comment:
I think an object would be suitable here. Allow me to update. :)

On Tue, Jan 27, 2015 at 12:37 PM, Giovanni Ortuno <notifications@github.com>
wrote:

> In
> android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/MetadataResolver.java
> <https://github.com/google/physical-web/pull/277#discussion_r23640517>:
>
> > @@ -90,13 +96,13 @@ public static void findUrlMetadata(final Context context, final MetadataResolver
> >     *
> >     * @param url The url for which to request data
> >     */
> > -  private static void requestUrlMetadata(String url) {
> > +  private static void requestUrlMetadata(String url, int txPower, int rssi) {
>
> What about making this three parameters an object. It seems you use all of
> them together all the time.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/physical-web/pull/277/files#r23640517>.
>